vevestaX/vevesta.py

```diff
@@ -288,10 +288,18 @@ def Y(self):
             
     @Y.setter
     def Y(self,value):
-        if value in self.__data.columns:
-            self.__Y=value
-        else:
-            print("Column not found")
+        if isinstance(value,Series):
+            if value.size==self.__sampleSize:
+                self.__Y=value
+                self.__v=None
```

Comment:
use __YcolumnName for __v. dont use names no one understands, it crucial step to write good code.